app/src/actions/vesselInfo.js

```diff
@@ -13,10 +14,14 @@ import {
   SET_PINNED_VESSEL_TITLE,
   TOGGLE_PINNED_VESSEL_EDIT_MODE,
   SET_RECENT_VESSEL_HISTORY,
-  LOAD_RECENT_VESSEL_HISTORY
+  LOAD_RECENT_VESSEL_HISTORY,
+  TIMELINE_MIN_INNER_EXTENT
```

Comment:
did you test this properly? you are importing TIMELINE_MIN_INNER_EXTENT from actions.js, but it's defined in constants.js